src/test/java/seedu/address/storage/JsonAdaptedPersonTest.java

```diff
@@ -13,18 +13,21 @@
 
 import seedu.address.commons.exceptions.IllegalValueException;
 import seedu.address.model.person.Address;
+import seedu.address.model.person.Company;
 import seedu.address.model.person.Email;
 import seedu.address.model.person.Name;
 import seedu.address.model.person.Phone;
 
 public class JsonAdaptedPersonTest {
     private static final String INVALID_NAME = "R@chel";
+    private static final String INVALID_COMPANY = "123 Company";
```

Comment:
wait why is this invalid, i thought company can be any string

Reply:
You're right, I goofed up the invalid vs valid

Reply:
Fixed with #[c497e37](https://github.com/AY2425S1-CS2103T-F13-4/tp/pull/43/commits/c497e37fbaf82119ae05330590f2b97a46dd88da)